### PR TITLE
B2B Enhancements

### PR DESCRIPTION
**Pull request by dobooth**
## Purpose of this pull request

This pull request (PR) updates a variety of pages to add B2B info and to enhance the find-ability of B2B resources. Fixes a couple bad links as well.

## Magento release version

-  2.4.x

## Product editions

Is this update specific to a product edition?

- Magento Commerce only

Is this update specific to an installed feature extension

- B2B extension

## Affected documentation pages

https://docs.magento.com/user-guide/quick-tour/build-loyalty.html
https://docs.magento.com/user-guide/quick-tour/self-service-tools.html
https://docs.magento.com/user-guide/quick-tour/b2b-quick-start.html
https://docs.magento.com/user-guide/stores/b2b-features.html
https://docs.magento.com/user-guide/configuration/general/b2b-features.html

### CODE REVIEW

File: src/quick-tour/build-loyalty.md [hunk 1]
```diff
@@ -21,3 +24,5 @@ Punch Out Solutions
 [1]: https://marketplace.magento.com/
 [2]: https://www.punchoutcatalogs.com/
 [3]: https://www.punchout2go.com/
+[4]: ../customers/account-dashboard-my-purchase-orders.html
```
Comment (leslietilling) [inline src/quick-tour/build-loyalty.md:27]: These do work, technically, but let's stick with our standard of {% link file/path.md %} for internal links.

File: src/quick-tour/self-service-tools.md [hunk 2]
```diff
@@ -6,6 +6,9 @@ title: Empower with Self-Service Tools
 Self-Service Tools
 :  Magento’s self-service model provides an [account dashboard]({% link customers/account-dashboard.md %}) that empowers customers to manage their own accounts and ordering process.
 
+Purchase Orders
+:  Allow companies to create Purchase Orders that can be controlled with company-defined Approval Rules.
```
Comment (leslietilling) [inline src/quick-tour/self-service-tools.md:10]: Not proper names
```suggestion
:  Allow companies to create purchase orders that can be controlled with company-defined approval rules.
```

File: src/quick-tour/build-loyalty.md [hunk 3]
```diff
@@ -3,6 +3,9 @@ b2b_only: true
 title: Build Loyalty
 ---
 
+Purchase Orders
+:  Use [Purchase Orders][4] and [Approval Rules][5] to allow purchasing according to your company's purchasing policies.
```
Comment (leslietilling) [inline src/quick-tour/build-loyalty.md:7]: Not proper names

```suggestion
:  Use [purchase orders][4] and [approval rules][5] to allow purchasing according to your company's purchasing policies.
```

File: src/quick-tour/b2b-quick-start.md [hunk 4]
```diff
@@ -34,9 +34,9 @@ sections:
     content: Give companies the convenience of charging purchases to their account, up to the credit limit that you determine for the company.
     url: /payment/payment-on-account.html
 
-  - title: Integration
-    content: Magento includes a flexible application programming interface (API) that integrates with a variety of ERP solutions from Magento partners.
-    url: /system/integrations.html
+  - title: Purchase Orders
+    content: Magento supports Purchase Orders, which can be controlled by Approval Rules.
```
Comment (leslietilling) [inline src/quick-tour/b2b-quick-start.md:38]: Not proper names

```suggestion
    content: Magento supports purchase orders, which can be controlled by approval rules.
```

File: src/quick-tour/build-loyalty.md (hunk 1 above)
Comment (dobooth) [reply src/quick-tour/build-loyalty.md]: Fixed

**Comment by dobooth** [comment]
running tests

**Pull request merged**